cvat/apps/iam/permissions.py

```diff
@@ -565,6 +584,15 @@ def create(cls, request, view, obj):
 
         return permissions
 
+    @classmethod
+    def create_scope_read(cls, request, storage_id):
+        try:
+            obj = CloudStorage.objects.get(id=storage_id)
+        except CloudStorage.DoesNotExist as ex:
+            raise ValidationError(str(ex))
+
+        return cls(**cls.unpack_context(request), obj=obj, scope=__class__.Scopes.LIST_CONTENT)
```

Comment:
Actually, it's neither `VIEW` nor `LIST_CONTENT` in their actual use, it's read and write depending on the situation. `View` doesn't fit because it's about the retrieving the storage configuration object. `List content` is the closest one from the existing (though it is also a bit different, because it supposes reading from the manifest file), but it is good enough to represent storage data access. I wouldn't like to introduce a new scope here to keep it simple. Probably, we can either remove `list content` and just use `view` everywhere (because now they are the same), or add 1 more scope like `access`.